Kernel/CommandLine.cpp

```diff
@@ -34,15 +36,24 @@ UNMAP_AFTER_INIT void CommandLine::initialize()
 {
     VERIFY(!s_the);
     s_the = new CommandLine(s_cmd_line);
+    dmesgln("Kernel Commandline: {}", kernel_command_line().string());
 }
 
-UNMAP_AFTER_INIT CommandLine::CommandLine(const String& string)
-    : m_string(string)
+UNMAP_AFTER_INIT void CommandLine::build_commandline(String cmdline_from_bootloader)
 {
-    s_the = this;
+    auto embedded_cmdline = StringView(s_embedded_cmd_line);
+    StringBuilder builder;
+    builder.clear();
+    builder.append(cmdline_from_bootloader);
+    if (!embedded_cmdline.is_null() && !embedded_cmdline.is_empty()) {
```

Comment:
`!is_empty()` implies `!is_null()`, so

```suggestion
    if (!embedded_cmdline.is_empty()) {
```